### ISSUE TITLE
NT8 FIX connection

### ISSUE THREAD

**Issue by b16aln** [FabienCarmagnac/NinjaTrader_FixBridge#2]

**Comment by wil70** [comment]
@b16aln where you able to make it work on NT8?

If I comprehend correctly, NinjaTrader is integrated with a broker such as IB for accessing market data and executing orders. This NT Strategy transforms NinjaTrader into an intermediate FIX server, enabling external FIX compatible programs to establish a connection with NinjaTrader through FIX to then access IB (or what ever broker NT is connected to). Is this understanding accurate?

Thanks

wil

cc: @FabienCarmagnac

**Comment by FabienCarmagnac** [comment]
Yes exactly. Not all fix protocol has been implemented  but works at least for futures and fx, certainly also for other products but I don't know the exact scope.
